cmd/git-sync/main.go

```diff
@@ -180,6 +193,7 @@ func main() {
 		}
 		if initialSync {
 			if *flOneTime {
+				log.V(2).Infof("synced one time, exiting", *flRev)
```

Comment:
This line doesn't seem to print a space between `"synced one time, exiting"` and `*flRev`